stdlib/public/runtime/Errors.cpp

```diff
@@ -132,12 +129,21 @@ static void dumpStackTraceEntry(unsigned index, void *framePC) {
   // We do not use %p here for our pointers since the format is implementation
   // defined. This makes it logically impossible to check the output. Forcing
   // hexadecimal solves this issue.
-  static const char *backtraceEntryFormat = "%-4u %-34s 0x%0.16lx %s + %td\n";
-
-  // Then dump the backtrace.
-  fprintf(stderr, backtraceEntryFormat, index, libraryName.data(),
-          foundSymbol ? symbolAddr : uintptr_t(framePC), symbolName.c_str(),
-          ptrdiff_t(uintptr_t(framePC) - symbolAddr));
+  // If the symbol is not available, we print out <unavailable> + offset
+  // from the base address of where the image containing framePC is mapped.
+  // This gives enough info to reconstruct identical debugging target after
+  // this process terminates.
+  if (foundSymbol) {
+    static const char *backtraceEntryFormat = "%-4u %-34s 0x%0.16lx %s + %td\n";
+    fprintf(stderr, backtraceEntryFormat, index, libraryName.data(), symbolAddr,
+          symbolName.c_str(), ptrdiff_t(uintptr_t(framePC) - symbolAddr));
```

Comment:
In C++ code, the line after the linebreak should be aligned with the opening paren.  (Here, `symbolName` should start in the same column as `stderr`.)